client/src/shared/planCard/index.jsx

```diff
@@ -7,53 +7,340 @@ import { Grid } from '@material-ui/core';
 import { useTheme } from '@material-ui/core/styles';
 import UnsafeRender from 'components/UnsafeRender';
 import { Text } from 'shared';
+import moment from 'moment'
+import styled from 'styled-components';
+import BookmarkOutlinedIcon from '../../assets/bookmark-outlined-icon.svg';
```

Comment:
Can you please create an index file for all the icons? So we import them all at once?

Reply:
done 14529a8a61e5337e4846adf586386e4e1bc3ae0c